### ISSUE TITLE
[Bear] Import data from `.bear2bk` produces unexpected notes

### ISSUE THREAD

**Issue by randy** [obsidianmd/obsidian-importer#224]
After sanitizing my Bear tags to remove spaces, I ran a second import into a new vault. Unfortunately, the affected notes contain proprietary information, so I cannot provide screenshots. However, I can give examples that may help.

The original note in Bear begins similar to this:

```
# JIRA-123 ❌
- [ ] Investigate
  * In some file, look at some thing...
  * second item, etc.

// many lines elided

#Task_List
```
So that first line is a "header 1" with an emoji, second line starts a to-do list, followed by an indented bullet list. After many lines, it ends with the tag "#Task_List" which is, let's say, my primary tag for all things "work." 

The import produced a note like that, as well as one like this:
```
# - Investigate
#Task_List
  * In some file, look at some thing...
  * second item, etc

// remainder of file as expected, but lacking the tag on the last line
```

In another example, the original note looks like this:
```
# 1:1
- [ ] item one
- [ ] item two
- [ ] etc
// many lines elided again
#today#
#Task_List
#Work_Dev
```

And the output looks like this:
```
# 11

# 1:1
- [ ] item one
- [ ] item two
- [ ] etc
/// many lines elided
#today#
#Task_List
#Work_Dev
```
There is no "correct" file that has the expected `1:1` title.

Let me know if I can provide any other information to assist on this. I'm looking to dump Bear because they pretty much told me I was wrong about some unexpected behavior their app has when moving lines up and down a todo/list. Obsidian handles the movements correctly.

**Comment by randy** [comment]
Did a little research and I suspect this is caused by some of the changes in #132, @tgrosinger. Unless I'm missing something, Bear always exports to files named by the H1 header. Backups might be a different story, but I can't make heads or tails of the output using `cat`. If you can point me in the right direction to parse a `.bear2bk` locally, I'd be happy to investigate further.

**Comment by tgrosinger** [comment]
Hi, thanks for the update. A `.bear2bk` file is just a zip containing textbundle files. You can unzip it to see the contained files that we iterate through with the importer.

If you can modify your `.bear2bk` by unzipping it, modifying the contents, and zipping it again you might be able to produce a file that reproduces the issue but isn't sensitive and could be shared with us. That would be very helpful to continue debugging the issue.

**Comment by randy** [comment]
Ok, well, this was eye-opening. A lot of the `.textbundle` files had unexpected names. I had to find some unique contents in them to find the actual notes they came from in Bear. When I found matches, I was also able to find a matching `.textbundle` with the "correct" filename. Upon further investigation, it turns out that the unexpected files appear to be from the trash! I'm not sure why these weren't ending up in the "trash" folder during the import, but I deleted them from Bear, ran a backup and imported that into a fresh vault. 

So far, everything appears to be okay; I just have to figure out how to deal with the change in tags behavior. Checking out plugins now. Closing this until I find something amiss!
